modules/understanding-upgrade-channels.adoc

```diff
@@ -7,116 +7,99 @@
+The service recommends only updates that have been tested and have no known serious regressions. For example, if your cluster is on {product-version}.1 and {product-title} suggests {product-version}.4, then it is safe for you to update from {product-version}.1 to {product-version}.4. Do not rely on consecutive patch numbers. In this example, {product-version}.2 is not and never was available in the channel, therefore updates to {product-version}.2 are neither recommended nor supported.
 
-ifndef::openshift-origin[]
-Update stability depends on your channel. The presence of an update recommendation in the `candidate-{product-version}` channel does not imply that the update is supported. It means that no serious issues have been found with the update yet, but there might not be significant traffic through the update to suggest stability. The presence of an update recommendation in the `fast-{product-version}` or `stable-{product-version}` channels at any point is a declaration that the update is supported. While releases will never be removed from a channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. When an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.
+[id="update-recommendation-removal"]
+== Update recommendation removals
+Red Hat monitors newly released versions and update paths associated with those versions both before and after they're added to supported channels. When a serious regression is identified Red Hat may remove affected update recommendations. If Red Hat chooses to remove update recommendations that action is taken in all relevant channels, simultaneously. The removal of recommended updates may happen either before or well after updates have been offered in supported channels.
+
+In the event that Red Hat removes update recommendations from any supported release we will eventually provide a superseding update paths to a future version which corrects the regression. There may however be a delay while the defect is corrected, tested, and promoted.
+
+When update recommendations are removed from supported channels those update paths remain supported but are no longer recommended. All supported but not recommended update paths updating from {product-title} version 4.10 and later will have a descriptive message and reference URL which links to more thorough explanation for the recommendation removal. Please review the linked reference and carefully consider whether you wish to continue with an update which is no longer recommended. See link:conditional-updates-overview_{context}[Conditional Updates] section for more information on this feature.
```

Comment:
It is not clear that we are talking about conditional updates in the beginning. We should say that in 4.10 or later versions we may conditionally recommend update paths. As part of the conditional update , the condition is evaluated and then CVO decided if the issue is applicable to the cluster or not.  Conditional update will  have a descriptive message and reference URL which links to more thorough explanation for the recommendation removal

Reply:
nit: the proposed text is talking about a singular risk, but a given A->B update may be exposed to several known risks, e.g. `4.10.30 -(blocked: StaleInsightsRunLevelLabel, StaleSELinuxPolicies)-> 4.11.3`.  But I expect folks will get the idea with either singular-risk or plural-risks phrasing.